code-challenges.test.js

```diff
@@ -14,38 +14,143 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+// describe("sentenceMaker", () => {
+//   it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized", () => {
+//     const people = [
+//       { name: "ford prefect", occupation: "a hitchhiker" },
+//       { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+//       { name: "arthur dent", occupation: "a radio employee" }]
+//       // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+//       expect(sentenceMaker(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
 
+//   })
+// })
+
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// ReferenceError: sentenceMaker is not defined
 
 // b) Create the function that makes the test pass.
 
+//Goal: Takes in an array of objects and returns an array with a sentence about each person with their name capitalized
+
+//Pseudocode:
+// Create a function expression called sentenceMaker. 
+// Create variable that invokes the map method on the array array parameter. (arraySentence)
+  // Convert array to string by invoking .join method. 
+ // Capitalize first letter of all names by invoking using string.charAt(0).toUpperCase() joint methods. 
+// Convert string back into array use .split method. 
+// Return desired out. 
+
+// const sentenceMaker= (arraySentence) => {
+//   arraySentence.map
+//   arraySentence.join
+//   arraySentence.charAt(0).toUpperCase()
+//   arraySentence.split
+// }
+
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// TypeError: arraySentence.charAt is not a function
+
+// ** NOTES TO INSTRUCTOR **: I know there's not alot of code but this one had me very confused spent multiple hours on it and stackoverflow /Google made me even more lost on how to approach the solution. 
```

Comment:
I can tell you put a lot of time and effort into this.  This is a hard one, for sure.  Your pseudocode is great - it really shows your thought process.  I hope the review in class helped explain a way to approach this question.  It looks your getting your methods a bit confused.  I encourage you to study the join and split methods.  Also, if you'd like to work through this problem together just let me know.